File: 003-Frontend/HomeTools.Front/src/app/dto/financas/cartoes/readCompraDto.ts
import { Categoria } from './../../../interfaces/categoria';

export interface ReadCompraDto {
  Id: number;
  Descricao: string;
  ValorInteiro: number;
  ValorCentavos: number;
  DataCompra: Date;
  QtdParcelas: number;
  CartaoId: number;
  Categoria: Categoria;
}

File: 002-Backend/LaPlata.API/LaPlata.Domain/DTOs/Cartoes/Compra/ReadCompraDTO.cs
namespace LaPlata.Domain.DTOs
{
    public class ReadCompraDTO
    {
        public int Id { get; set; }
        public string Descricao { get; set; }
        public int ValorInteiro { get; set; }
        public int ValorCentavos { get; set; }
        public DateTime DataCompra { get; set; }
        public int QtdParcelas { get; set; }
        public bool Paga { get; set; }
        public int CartaoId { get; set; }
        public ReadCategoriaDTO Categoria { get; set; }
    }
}
